docs/reporting.rst

```diff
@@ -0,0 +1,164 @@
+  the amount of variance explained by the component (larger dot,
+  larger variance).
+.. image:: /_static/rep01_kapparhoScatter.png
+  :align: center
+  :height: 400px
+
+* **Kappa Scree Plot:** This scree plots provides a view of the components ranked by `Kappa`
+  As in the previous plot, each dot represents a component. The color of the dot inform us
+  about classification status. In this plot, size is not related to variance explained.
+.. image:: /_static/rep01_kappaScree.png
+  :align: center
+  :height: 400px
+
+* **Rho Scree Plot:** This scree plot provides a view of the components ranked by `Rho`.
+  As in the previous plot, each dot represents a component. The color of the dot informs us
+  about classification status. In this plot, size is not related to variance explained.
+.. image:: /_static/rep01_rhoScree.png
+  :align: center
+  :height: 400px
+
+* **Variance Explained Plot:** This pie plot provides a summary of how much variance is explained
+  by each individual component, as well as the total variance explained by each of the three
+  classification categories (i.e., accepted, rejected, ignored). In this plot, each component is
+  represented as a wedge, whose size is directly related to the amount of variance explained. The
+  color of the wedge inform us about the classification status of the component. For this view,
+  components are sorted by classification first, and inside each classification group by variance
+  explained.
+.. image:: /_static/rep01_varexpPie.png
+  :align: center
+  :height: 400px
+
+Individual Component View 
+-------------------------
+This view provides detailed information about an individual 
+component (selected in the summary view, see below). It includes two different
+interactive plots.
+
+* **Time series:** This plot shows the time series associated with a given component
+  (selected in the summary view). The x-axis represents time (in units of TR), and the
+  y-axis represents signal levels (in arbitrary units). Finally, the color of the trace
+  informs us about the component classification status.
+
+.. image:: /_static/rep01_tsPlot.png
+  :align: center
+  :height: 150px
+
+* **Spectrum:** This plot shows the spectrogram associated with a given component
+  (selected in the summary view). The x-axis represents frequency (in Hz), and the
+  y-axis represents spectral amplitude. Finally, the color of the trace
+  informs us about the component classification status.
+
+.. image:: /_static/rep01_fftPlot.png
+  :align: center
+  :height: 150px
+
+.. note::
+  When a report is initially loaded, as no component is selected on the
+  summary view, the individual component view appears empty.
+
+Reports User Interactions
+=========================
+
+As previously mentioned, all plots in the report allow user interactions. The list of permitted
+interactions vary by plot, but can be easily infered by the toolbar that accompanies each plot
```

Comment:
In the current view, the toolbar is only on the Kappa-Rho plot, so we likely need to rephrase this section accordingly.

Reply:
[1b59407](https://github.com/ME-ICA/tedana/pull/465/commits/1b594074fdfa0fcc48f6cd28caf4fd3e397b13c3) should take care of this.